ansible/roles/debops.discourse/defaults/main.yml

```diff
@@ -0,0 +1,686 @@
+---
+# .. vim: foldmarker=[[[,]]]:foldmethod=marker
+
+# debops.discourse default variables [[[
```

Comment:
You should remove the outermost `[[[` and `]]]` markers, so that the defaults file has one less level of folding we need to open in `vim`.